WALMA-15: Adding GoToContentItemDisplayByIdAsync

dministro opened the pull request:
WALMA-15

dministro committed:
Adding GoToContentItemDisplayByIdAsync

dministro committed:
Adding user, role and permission related shortcuts

dministro committed:
Updating HL NuGet package version

dministro committed:
Adding workflows related shortcut

dministro committed:
Updating Lombiq.HelpfulLibraries.* Nuget package version

Luko6 commented on Lombiq.Tests.UI.Shortcuts/Controllers/SecurityController.cs:
```diff
@@ -0,0 +1,79 @@
+        return Ok("Success");
+    }
+
+    private async Task<bool> IsPermissionExistsAsync(string permissionName) =>
```
Rename to `PermissionExistsAsync`

Luko6 commented on Lombiq.Tests.UI.Shortcuts/Controllers/SecurityController.cs:
```diff
@@ -0,0 +1,79 @@
+        return Ok("Success");
+    }
+
+    public async Task<IActionResult> AllowPermissionToRole(string permissionName, string roleName)
```
Rename to `AddPermissionToRole`

Luko6 commented on Lombiq.Tests.UI.Shortcuts/Controllers/WorkflowsController.cs:
```diff
@@ -0,0 +1,45 @@
+        _workflowTypeStore = workflowTypeStore;
+    }
+
+    public async Task<IActionResult> HttpEventGenerateUrl(string workflowTypeId, string activityId, int tokenLifeSpan)
```
Rename to `GenerateHttpEventUrl`

dministro committed:
Renaming AllowPermissionToRole to AddPermissionToRole
Renaming IsPermissionExistsAsync to PermissionExistsAsync
Renaming HttpEventGenerateUrl to GenerateHttpEventUrl

Luko6 commented on Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs:
```diff
@@ -85,6 +85,31 @@ public static async Task<string> GetCurrentUserNameAsync(this UITestContext cont
+    /// <summary>
+    /// Allows a permission to a role.
+    /// </summary>
+    public static Task AllowPermissionToRoleAsync(this UITestContext context, string permissionName, string roleName) =>
```
Rename this too similarly like the method called.

Luko6 commented on Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs:
```diff
@@ -224,4 +249,20 @@ public interface IShortcutsApi
+    /// Retrieves URI for a <see cref="OrchardCore.Workflows.Http.Activities.HttpRequestEvent"/> in a workflow. The
+    /// target app needs to have <c>Lombiq.Tests.UI.Shortcuts.Workflows</c> enabled.
+    /// </summary>
+    public static async Task<string> WorkflowsHttpEventGenerateUrlAsync(
```
Rename this too similarly like the method called.

dministro committed:
Renaming AllowPermissionToRoleAsync to AddPermissionToRoleAsync
Renaming WorkflowsHttpEventGenerateUrlAsync to GenerateHttpEventUrlAsync
